backend/lib/models/Author.js

```diff
@@ -2,6 +2,19 @@
 const { Schema, model, ObjectId } = require("mongoose");
 const { schema: locationSchema } = require("./Location");
 
+const USER_TYPES = [
```

Comment:
I do like this approach, then you can reference like  `default: LOCATION_TYPES.POINT`  instead of `default: "Point",` when you need to use a specific value in logic. 

But I might file this one under future refactoring improvement and go with this for now?